test/controllers/concerns/authentication_test.rb

```diff
@@ -112,7 +112,12 @@ def raising
 
   context 'after successful login' do
     setup do
-      stub_rbac_permissions(Rbac::COMPLIANCE_VIEWER, Rbac::INVENTORY_VIEWER)
+      stub_rbac_permissions(
+        [
+          { permission: Rbac::COMPLIANCE_VIEWER },
+          { permission: Rbac::INVENTORY_VIEWER }
+        ]
+      )
```

Comment:
The `stub_rbac_permission` method signature should not change. The old way of calling it with a simple array should work as before. In addition it could handle the complex values...